core/jobs/batch_jobs/suggestion_edit_state_content_deletion_jobs_test.py

```diff
@@ -0,0 +1,280 @@
+
+    jobs = suggestion_edit_state_content_deletion_jobs
+
+    JOB_CLASS: Type[
+        jobs.AuditDeprecatedSuggestionEditStateContentModelsDeletionJob
+    ] = jobs.AuditDeprecatedSuggestionEditStateContentModelsDeletionJob
+
+    def create_suggestion(
+            self,
+            suggestion_type: str,
+            target_type: str,
+            target_id: str,
+            author_id: str,
+            final_reviewer_id: str,
+            language_code: None | str
+            ) -> suggestion_model:
+        """Creates new suggestion.
+
+        Args:
+            suggestion_type: str. Type of the suggestion that is being
+                created.
+            target_type: str. Type of traget entity for which the suggestion
+                is being created.
+            target_id: str. The ID of the target entity.
+            author_id: str. The ID of user who creates the suggestion.
+            final_reviewer_id: str. The user ID of the reviewer.
+            language_code: None | str. Code of the language in which the
+                suggestion is created.
+
+        Returns:
+            GeneralSuggestionModel. Returns newly created
+            GeneralSuggestionModel.
+        """
+
+        return self.create_model(
+            suggestion_model,
+            suggestion_type=suggestion_type,
+            target_type=target_type,
+            target_id=target_id,
+            target_version_at_submission=1,
+            status='accepted',
+            author_id=author_id,
+            final_reviewer_id=final_reviewer_id,
+            change_cmd={},
+            score_category=(
+                suggestion_models.SCORE_TYPE_TRANSLATION +
+                suggestion_models.SCORE_CATEGORY_DELIMITER + 'English'),
+            language_code=language_code
+        )
+
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.edit_state_content_suggestion_model_1: suggestion_model = (
+            self.create_suggestion(
+                feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT,
+                feconf.ENTITY_TYPE_EXPLORATION,
+                'target_1',
+                'author_1',
+                'reviewer_1',
+                None
+            ))
+
+        self.edit_state_content_suggestion_model_2: suggestion_model = (
+            self.create_suggestion(
+                feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT,
+                feconf.ENTITY_TYPE_EXPLORATION,
+                'target_2',
+                'author_2',
+                'reviewer_2',
+                None
+            ))
+
+        self.create_translation_suggestion_model: suggestion_model = (
+            self.create_suggestion(
+                feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+                feconf.ENTITY_TYPE_EXPLORATION,
+                'target_3',
+                'author_3',
+                'reviewer_3',
+                'hi'
+            ))
+
+        self.add_question_suggestion_model: suggestion_model = (
```

Comment:
```suggestion
        self.question_suggestion_model: suggestion_model = (
```

In general, avoid starting variable names with verbs.

Reply:
done